docs/source/nodeoperations/replace_node.md

```diff
@@ -68,4 +70,37 @@ _This procedure is for replacing one dead node. To replace more than one dead no
    UN  10.43.231.189  91.03 KB   256          ?       35d0cb19-35ef-482b-92a4-b63eee4527e5  us-east-1a
    UN  10.43.191.172  74.77 KB   256          ?       1ffa7a82-c41c-4706-8f5f-4d45a39c7003  us-east-1a
    ```
-1. Run the repair on the cluster to make sure that the data is synced with the other nodes in the cluster. You can use [Scylla Manager](manager.md) to run the repair.
+1. Run the repair on the cluster to make sure that the data is synced with the other nodes in the cluster. 
+   You can use [Scylla Manager](manager.md) to run the repair.
+
+### Replacing a dead seed node
+
+1. Degrade seed node to non-seed node. This really doesn't do anything at this point, except Operator will treat this pod differently.
+    ```
+    kubectl -n scylla label svc <pod-name> scylla/seed-
+    ```
+
+1. Mark pod with replace label to tell Operator that this node must be replaced.
+    ```
+    kubectl -n scylla label svc <pod-name> scylla/replace=""
+    ```
+   Operator will remove and recreate pod, service and pvc. Seed nodes are hardcoded (first two pods in rack), 
+   the recreated service will contain a seed label.
+   This causes Scylla to fail to start because seed nodes cannot be replaced.
+
+1. Remove the seed label again, and manually delete the pod, in order to allow the sidecar to pick up the state we want.
```

Comment:
Wonder if we could avoid the double unlabel + double delete with:
1. degrade/unlabel
2. delete the PVC
3. delete the Pod
4. mark it as a seed again
5. rolling restart

Reply:
We would have to have #507, because SVC would keep the same IP and new (empty) storage would be attached.